plugin/mocha.vim

```diff
@@ -127,7 +127,16 @@ endfunction
 
 " Current Spec File Name
 function! InSpecFile()
-  return match(expand("%"),'\v(.js|.coffee)$') != -2
+  " Not a js or coffee file
+  if match(expand('%'), '\v(.js|.coffee)$') == -1
+    return 0
+  endif
+
+  " Check for describe block
+  let l:contents = join(getline(1,'$'), "\n")
+  let l:jsRegex = '\v<describe(.\w+)?\s{-}\(.{-}[''"].{-}[''"]\s{-}, {-}function\s{-}\(.{-}\)'
+  let l:coffeeRegex = '\v<describe\s+[''"].{-}[''"]\s{-},\s+-\>'
+  return match(l:contents, l:coffeeRegex) != -1 || match(l:contents, l:jsRegex) != -1
```

Comment:
Done. Is [this](https://github.com/yannvanhalewyn/vim-mocha/commit/80ab6726000192a6a783f675ff3de90d21e377e1) better? I'm actually not sure it is, I can imagine people having their own describe function that takes a string.